### PR TITLE
Blindness fix

### PR DESCRIPTION
**Pull request by Bod9001**
the strategy is the brain give blindness, 
and the eyes give blindness prevention

so,when you leave your body you're losing the blindness from your brain, and the blindness protection from the eyes

but when you remove your eyes you're losing the blindness protection so you become blind

### Changelog:

CL: [Fix] Fixes blindness not being applied when eyes are removed

### CODE REVIEW

**Comment by corp-0** [comment]
what happens when you lose your brain?

**Comment by Q4ik** [comment]
Death if not reimplanted. Doesn’t fave to be the head either.

File: UnityProject/Assets/Scripts/Managers/AudioManager.cs [hunk 1]
```diff
@@ -25,6 +25,23 @@ public class AudioManager : SingletonManager<AudioManager>
         public AudioMixerGroup AmbientMixer;
         public AudioMixerGroup TTSMixer;
 
+        private float GameplayVolumeLevel = 1;
+
+        public float gameplayVolumeLevel
+        {
+	        set
+	        {
+		        if (value > 1) //No earap please
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/AudioManager.cs:34]: lmao what happened that made you do this?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/AudioManager.cs:34]: There has been two reported instances of the jukebox having its audio above 100%

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Managers/AudioManager.cs:34]: well, going to work on ears, e.g being in space Makes it so you can't hear stuff

**Pull request merged**